lms/templates/courseware/course_about.html

```diff
@@ -167,8 +167,9 @@
             reg_element_id = "reg_then_add_to_cart"
           %>
           <a href="${reg_href}" class="add-to-cart" id="${reg_element_id}">
-            ${_("Add {course_name} to Cart ({price})")\
-              .format(course_name=course.display_number_with_default, price=course_price)}
+            ${_("Add {course_name} to Cart")\
+              .format(course_name=course.display_number_with_default)}
+            <div>${_("({price} USD)").format(price=course_price)}</div>
```

Comment:
Please use `<span>` as an inline element instead of `<div>`. http://www.w3schools.com/html/html_blocks.asp
Also make these 3 lines a single line.